src/functionalTest/java/uk/gov/hmcts/reform/ccd/utils/TaskDeleteTestUtils.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.ccd.utils;
+
+import jakarta.inject.Inject;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.helper.S2SHelper;
+import uk.gov.hmcts.reform.ccd.helper.SecurityUtils;
+import uk.gov.hmcts.reform.ccd.parameter.ParameterResolver;
+import uk.gov.hmcts.reform.ccd.util.log.TasksDeletionRecordHolder;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.awaitility.Awaitility.with;
+
+@Component
+public class TaskDeleteTestUtils {
+
+    @Inject
+    private TasksDeletionRecordHolder tasksDeletionRecordHolder;
+
+    @Inject
+    private SecurityUtils securityUtils;
```

Comment:
I don't see any usage of securityUtils, parameterResolver, s2SHelper and fileUtils

Reply:
These are now removed